src/plugins/intel_gpu/src/graph/lstm.cpp

```diff
@@ -48,7 +48,7 @@ std::string lstm_inst::to_string(lstm_node const& node) {
     lstm_info.add("weights id", weights_id);
     lstm_info.add("recurrent id", recurrent_id);
     lstm_info.add("bias id", bias_id);
-    lstm_info.add("peepholes id", peepholes_id);
+    lstm_info.add("peepholes id", std::move(peepholes_id));
```

Comment:
Does it make sense to move small strings? And if it makes sense, then why `peepholes_id` moved only?
Same question for other changes in `to_string(..)` method